impl-openssl/build.rs

```diff
@@ -11,6 +11,9 @@ fn main() {
         Ok(ref v) if v == "110" => {
             println!("cargo:rustc-cfg=has_alpn");
         }
-        _ => panic!("Unable to detect OpenSSL version"),
+        Ok(ref v) if v == "111" => {
```

Comment:
This should probably be changed to accept any version above "102" because it's unlikely that they are going to undo alpn support. But it's OK to keep it as is.

Reply:
@stepancheg  Resolved.